src/layers/logging.rs

```diff
@@ -174,8 +316,14 @@ impl Accessor for LoggingAccessor {
             .map(|v| {
                 debug!(
                     target: "opendal::services",
-                    "service={} operation={} path={} offset={:?} size={:?} -> got reader", self.scheme, Operation::Read, args.path(),args.offset(),  args.size() );
-                v
+                    "service={} operation={} path={} offset={:?} size={:?} -> got reader",
+                    self.scheme,
+                    Operation::Read,
+                    args.path(),
+                    args.offset(),
+                    args.size()
+                );
+                LoggingReader::make_reader(self.scheme, Operation::Read, args.path(), args.size(), v)
```

Comment:
Maybe we don't need to input the `size`.

Reply:
Why? I think recording the size of reading or writing is good for tracing.🤔

Reply:
Hey, this is logging. Printing the size every time in the log seems useless.

Reply:
Sorry didn't notice that, I'll remove this field.